products/nvidiaproducts.md

```diff
@@ -12,6 +12,18 @@ releaseDateColumn: true
 discontinuedColumn: true
 releaseColumn: false
 releases:
+-   releaseCycle: "Professional Ada Lovelace (AD1xx)"
+    support: true
+    eol: false
+    discontinued: false
+
```

Comment:
Why the empty lines before releaseDate and not after ?

Reply:
is it ok now ? @marcwrobel

Reply:
Yes. I saw the same formatting in other files and I wondered if it was done on purpose.

Reply:
@marcwrobel actually it was caused by a bug from our automation script